### ISSUE TITLE
[RFC] Changelog field

### ISSUE THREAD

**Issue by Duncaen** [voidlinux/void-packages#14136]
For the new changelog field we need to increase the required version of xbps in `xbps-src`.
This was done yesterday but I reverted the changes  (5334409c12e56e429fc53dcb72d4aedb2495dfc2, 94ed33657b8a0d77a82099057b00fa4a54d48ec5) because there is at least one major issue and some policy that wasn't discussed before.

Travis uses static xbps binaries from http://repo.voidlinux.eu/static/ which we need to update before we depend on a newer version.

Another small style/documentation issue is that we need to define what kind of changelog links are allowed or not.

The change from yesterday said it MUST be plaintext, I think that it is too limiting and all forms of changelog URLs should be allowed. I think plaintext can be preferred but shouln't be required.

**Comment by chneukirchen** [comment]
We need to roll new static tarballs for 0.52 anyway, no?

And yes, it should be any URL.

**Comment by Duncaen** [comment]
Yes we need new static tarballs anyways, but we need to decide who or which key should sign the checksums.
Travis doesn't verify the signatures, we can already do the static tarballs and sign them later.

**Comment by maxice8** [comment]
Another thing to define, preference for changelog from master branch that might include changes not in the package or (if available) for the most recent version packaged using $version ?

**Comment by pullmoll** [comment]
I agree: plaintext should be preferred and not mandatory for obvious reasons.

For xbps-0.52 itself it seems some packages end up in `<arch>-stagedata`. It happened for the update of `qtcreator-4.6.1` and I don't see a possible reason in `xbps-create(1)`.  I think the how and why should be documented, no?

**Comment by maxice8** [comment]
are debian changelogs ok ? for `xapps` they are the only option for plaintext logs and are available on the main repo under debian/changelog.

or should it be pointed to the github release page of xapps.

**Comment by chneukirchen** [comment]
`debian/changelog` is ok if it's written by upstream.

**Comment by maxice8** [comment]
Is there a specific place where changelog= should go ?

**Comment by chneukirchen** [comment]
The existing `_changelog` are directly below `homepage`.
